pkg/awsclient/elb_helper.go

```diff
@@ -56,14 +62,16 @@ func (c *awsClient) CreateClassicELB(elbName string, subnets []string, listenerP
 
 // SetLoadBalancerPrivate sets a load balancer private by removing its
 // listeners (port 6443/TCP)
-func (c *awsClient) SetLoadBalancerPrivate(elbName string) error {
+func (c *AwsClient) SetLoadBalancerPrivate(elbName string) error {
```

Comment:
NIT. I think this function name is confusing.  Setting something private, to me, is making it go from external to internal.  This is removing the listeners.  Maybe we renamed this function to align with what it does?

Reply:
I appreciate that. Background reasoning follows - The reason I chose to do it this way is that I wanted the controller to set the stage for a common cloud interface centred around what it means to stop traffic from getting to the machines backing the load balancer. The way we go about it for AWS or GCP are necessarily different, and for AWS, one way for this operator to stop the flow of traffic is to remove the listeners. Removing listeners seems to me to be least destructive.

I also agree that as the scope of the operator has expanded that perhaps it's not named the best. Maybe I can rename it to be `RemoveLoadBalancerListeners` and the higher order interface can still use the words Private/Public and let the implementation details to the cloud libraries.